src/django/api/templates/mail/api_limit_body.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            Open Apparel Registry contributor API limit notice
+        </title>
+    </head>
+    <body>
+        <p>
+            Hi,
+        </p>
+        <p>
+            You're receiving this email because you have used {{ count }} of
+            your {{ limit }} requests for this month.
+        </p>
+        {% if grace_limit %}
+        <p>
+            The grace limit previously set was {{ grace_limit }}.
```

Comment:
While testing I realized that we are being more detailed about our limit calculations than we absolutely have to be.

Consider this simplified version that excludes the current count
```
        {% if grace_limit %}
        <p>
            You're receiving this email because you have exceeded
            your limit of {{ grace_limit }} OAR API requests for this month.
        </p>
        {% else %}
        <p>
            You're receiving this email because you have exceeded
            your limit of {{ limit }} OAR API requests for this month.
        </p>
        {% endif %}
```